dfusion_rust_core/src/models/standing_order.rs

```diff
@@ -69,35 +110,199 @@ impl From<mongodb::ordered::OrderedDocument> for StandingOrder {
     }
 }
 
+impl From<Arc<Log>> for StandingOrder {
+    fn from(log: Arc<Log>) -> Self {
+        let mut bytes: Vec<u8> = log.data.0.clone();
+        // Get basic data from event
+        let batch_index = U256::pop_from_log_data(&mut bytes);
+        let valid_from_auction_id = U256::pop_from_log_data(&mut bytes);
+        let account_id = u16::pop_from_log_data(&mut bytes);
+        // let packed_orders_bytes = bytes;
+
+        println!("Parsing orders from packedOrders. {} bytes. {:?}", bytes.len(), bytes);
+        assert!(bytes.len() % 26 == 0, "Each order should be packed in 26 bytes");
```

Comment:
I guess these two lines are not super good practice.
Not sure what's the best way to log messages. I added that cause it was crashing the assert, for some reason I'm getting a different number of bytes there.

The `assert` I guess it's also not beautiful because it makes the code to "panic" hehe
I saw the usual approach is to use the trait `Result` but it just didn't see how it fits here. Maybe, the validations should be done before calling this function, not sure.